.github/ISSUE_TEMPLATE/1.bug_report.yaml

```diff
@@ -109,6 +117,47 @@ body:
         - Not Applicable
     validations:
       required: true
+  
+  - type: dropdown
+    attributes:
+      label: Frontend framework
+      description: Which frontend framework is your app built upon?
+      multiple: false
+      options:
+        - "Angular"
+        - "Gatsby"
+        - "Hugo"
+        - "JavaScript"
+        - "Jekyll"
+        - "Nextjs"
+        - "React"
+        - "Remix"
+        - "Svelte"
+        - "Vite"
+        - "Vue"
+    validations:
```

Comment:
maybe have an option for "Other" since this is a required field

Reply:
Done!